.github/workflows/release.yml

```diff
@@ -4,13 +4,22 @@ on:
   push:
     branches:
       - main
```

Comment:
@agoose77 if we want to keep testing here, we could change this to `pypi` and switch it back before squashing.